plugins/inputs/minecraft/minecraft.go

```diff
@@ -0,0 +1,144 @@
+package minecraft
+
+// minecraft.go
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+const sampleConfig = `
+## server address for minecraft
+server = "localhost"
+## port for RCON
+port = "25575"
+## password RCON for mincraft server
+password = "replace_me"
```

Comment:
This bit does need two space indents so that it looks right in `telegraf config`